### PR TITLE
Deploy App Store to Prod

### PR DESCRIPTION
**Pull request by annaleeswitek**
# Description

This PR is to promote the App Store feature to production. 

## Clickup Ticket

[App Store: Refine UI for running Events on a File or Files ](https://app.clickup.com/t/8685ncb85)

## Type of change

- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Validated this branch locally. 

# Checklist:

_Delete those that don't apply. Only apply the final commit message using the changelog when merging a feature to `DEVELOPMENT` that will be deployed to all users._

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have added unit tests that prove my fix is effective or that my feature works
- [X] I have pushed the final commit message using the [changelog format](https://github.com/Pennsieve/pennsieve-app/wiki/CHANGELOG)

### CODE REVIEW

File: src/components/Integrations/integration-list-item/IntegrationListItem.vue [hunk 1]
```diff
@@ -1,111 +1,72 @@
...
+      <!--      <el-col-->
+      <!--        :sm="8"-->
+      <!--      >-->
+
+      <!--      </el-col>-->
+      <!--      <el-col-->
+      <!--        :sm="8"-->
+      <!--      >-->
+      <!--        {{ created }}-->
+      <!--      </el-col>-->
```
Comment (edmore) [inline src/components/Integrations/integration-list-item/IntegrationListItem.vue:16]: @annaleeswitek can we remove as opposed to commenting this out?

File: src/components/Integrations/applicationsList/ApplicationsList.vue [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+              target="_blank"
+            >
+              What's this?
+            </a>
+          </p>
+        </div>
+
+        <div class="reg-button">
+          <bf-button v-if="hasAdminRights" @click="openAddIntegration">
+            Register Application
```
Comment (edmore) [inline src/components/Integrations/applicationsList/ApplicationsList.vue:23]: When I click on "Register Application", I still see webhook references, for example some api URL suggestions.
We also still have the generic "Add Integration" titles/text there as well. Do we plan to keep it that way?

Comment (annaleeswitek) [reply src/components/Integrations/applicationsList/ApplicationsList.vue:23]: @edmore Thanks for noticing this - let me fix this before we deploy this code to prod.

File: src/components/Integrations/integration-list-item/IntegrationListItem.vue (hunk 1 above)
Comment (annaleeswitek) [reply src/components/Integrations/integration-list-item/IntegrationListItem.vue:16]: @edmore - sure

**Pull request closed without merging**